shared/naturalcrit/codeEditor/helpers/widget-elements/constants.js

```diff
@@ -0,0 +1,30 @@
+export const UNITS = ['cm', 'mm', 'in', 'px', 'pt', 'pc', 'em', 'ex', 'ch', 'rem', 'vw', 'vh', 'vmin', 'vmax', '%'];
+
+export const HINT_TYPE = {
+	VALUE         : 0,
+	NUMBER_SUFFIX : 1
+};
+
+export const WIDGET_TYPE = {
```

Comment:
I think we may need to step back and make sure our definitions are all the same.

To me a `widget` is one of these: 
![image](https://github.com/naturalcrit/homebrewery/assets/5878534/b871c056-e450-4588-9e6e-57e9e454b062)

But this code indicates that the widget is this:
![image](https://github.com/naturalcrit/homebrewery/assets/5878534/833876f6-8f01-4491-a904-3b011256b69d)

Reply:
Fair, that should be `SNIPPET_TYPE`

Reply:
Ok, in that case could we rename these types to: BLOCK, INLINE, and INJECTOR? Since the single { we use for images can be applied to other elements to.